Resources/Prototypes/Reagents/Materials/metals.yml

```
@@ -21,28 +21,28 @@
   unit: materials-unit-bar
   icon: { sprite: Objects/Materials/ingots.rsi, state: silver }
   color: "#C0C0C0"
-  price: 0.15
+  price: 1.5
 
 - type: material
   id: Brass
   stackEntity: SheetBrass1
   name: materials-brass
   icon: { sprite: Objects/Materials/Sheets/metal.rsi, state: brass }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)